src/renderer/store/editor.js

```diff
@@ -24,7 +24,8 @@ const state = {
     word: 0,
     character: 0,
     all: 0
-  }
+  },
+  platform: ''
```

Comment:
platform:  process.platform

Set the init value, So, you don't need to dispatch the action 'GET_PLATFORM'.